apps/slack/src/lib/slack.ts

```diff
@@ -12,6 +12,14 @@ export const sendSlackMessage = async (
     order: { id, number, user, shippingAddress, subtotal, shippingPrice, total },
   } = data;
 
+  const getCustomerName = () => {
```

Comment:
WDYT about falling back to the user email? It should be available in all of the orders, even without the account